test/integration/cypress/support/page_files/NodePage.ts

```diff
@@ -7,23 +7,46 @@ class EdgeList {
     return cy.get(this.selector + " [data-cy=edge]");
   }
 
+  get list() {
+    return cy.get(this.selector);
+  }
+
   get title() {
     return cy.get(this.selector + " [data-cy=edge-list-title]");
   }
 }
 
+export enum NodePageTab {
+  PredicateGrid = "predicate-grid",
+  PredicateList = "predicate-list",
+}
+
 export class NodePage extends Page {
   constructor(private readonly nodeId: string) {
     super();
   }
 
+  assertTabSelected(tab: NodePageTab) {
+    return this.getTab(tab).should("have.class", "Mui-selected");
+  }
+
   get datasource() {
     return cy.get(this.frame.selector + " [data-cy=node-datasource]");
   }
 
-  edgeList(predicate: string) {
+  getTab(tab: NodePageTab) {
```

Comment:
We haven't been prefixing page object accessors with verbs ("get"), and you don't elsewhere.